public/app/plugins/panel/heatmap-new/plugin.json

```diff
@@ -2,10 +2,10 @@
   "type": "panel",
   "name": "Heatmap (preview)",
```

Comment:
Should we update name as in this step? Renan the old one to Heatmap (legacy) and this to Heatmap (new) or something like that?

Reply:
I'll change this to "Heatmap (new)" -- it seems a bit extreme to call the other one legacy if we are just going to auto swap it in the next release